lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_backmerge_pull_request_action.rb

```diff
@@ -110,17 +110,58 @@ def self.create_backmerge_pr(token:, repository:, title:, head_branch:, base_bra
 
         other_action.push_to_git_remote(tags: false)
 
-        pr_body = <<~BODY
-          Merging `#{head_branch}` into `#{base_branch}`.
+        # Live playground to edit this graph
+        # https://mermaid.live/edit#pako:eNqNkU1PwzAMhv9KZanqpWVFwCVHQOKCxoFrLl7itdGaZEoTTWjqf8eLOo2i8eGTPx77TewjKK8JBJTlUbqCzTgTRTEHJ6s6E18C7vtqkc4li8Y9BnSqX6MlBqoYkttV9Y_cW9AUGLxt2_Y7Nfb-8OStNfEVNzQwtcVhpCU1XUJ2p7KU7vzAXNlkmSLQQDjS6v7m4S7nVU9q51O8UsmSv7jzSEuho9Xc3pzaG-Oib_KXlxr_QL8onbuuVy9unvrXbKiBCV645qvme0mIPVmSINjVtMU0RAm8O0YxRf_-4RQIbqca0l5jpGeDXUALIu9_-gRejKzU
+        meramid_git_graph = <<~GRAPH
+          ```mermaid
+          %%{
+            init: {
+              'gitGraph': {
+                'mainBranchName': '#{base_branch}',
+                'mainBranchOrder': 1000,
+                'showCommitLabel': false
+              }
+            }
+          }%%
+          gitGraph
+             branch #{head_branch}
+             checkout #{head_branch}
+             commit
+             commit
+             commit
+             branch #{intermediate_branch}
+             checkout #{intermediate_branch}
+             commit
+             checkout #{base_branch}
+             commit
+             commit
+             merge #{base_branch}
+          ```
+        GRAPH
 
-          Via intermediate branch `#{intermediate_branch}`, to help fix conflicts if any:
+        ascii_git_graph = <<~GRAPH
           ```
           #{head_branch.rjust(40)}  ----o-- - - -
           #{' ' * 40}       \\
           #{intermediate_branch.rjust(40)}        `---.
           #{' ' * 40}             \\
           #{base_branch.rjust(40)}  ------------x- - -
           ```
+        GRAPH
+
+        pr_body = <<~BODY
+          Merging `#{head_branch}` into `#{base_branch}`.
```

Comment:
wdyt about adding the chart type as a parameter? Something like `:image`, `:ascii`, `:both`.

Reply:
My first instinct was to say thatI'd rather always have both in case the Mermaid one fails, which is something we cannot know at the time we generated the body because it depends on GitHub.

But, it's good to build flexible systems (_unopinionated tools, opinionated golden path_) and the adding a parameter like this does only makes the code a tiny bit more complex (_simple is usually better_).

I'll update 👍

Reply:
> I'll update 👍

Spawned #596 to not slow this down while waiting for that.